### DISCUSSION TITLE
Adding Middle Name Option

### DISCUSSION THREAD

**Discussion by amdeis** [knowledgefutures/pubpub#3041] [Ideas]
**Request a Middle Name option** in the profile of PubPub accounts (maybe have it be "optional" instead of "required," though). Authors of our journal often inquire about how to add a middle initial to their name in the byline of articles if the have PubPub accounts. Currently the only option that processes correctly is having them add a space, middle initial, and period after it IN THE FIRST NAME space. 

We've also had authors add the middle initial or middle name before their LAST name, which appears OK in the byline but does not format correctly in the auto-cite button at the top right of articles. 

![Screenshot 2024-03-13 at 16-42-29 Amara M  Deis · PubPub](https://github.com/pubpub/pubpub/assets/117661410/d084f400-3a34-4e98-8328-f4e3140abf23)

**Comment by meeloun** [comment]
When choosing online course outsourcing       [http://www.hotessay.net/our_service5.php](http://www.hotessay.net/our_service5.php)      services, it is necessary to pay attention to multiple aspects to ensure that your choice is correct. By carefully identifying and communicating with institutions, it can help you find personalized services that truly meet your needs.

**Comment by amdeis** [comment]
Just noting another handful of emails from authors over the last month of new users asking how to add their middle name. We have started to include instructions for authors on how to include their middle names when they create an account to use our platform. Hope this is a feature able to be added one day!
